src/lib/tuning/abstract_evaluator.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "tuning/tuning_choice.hpp"
+
+namespace opossum {
+
+/**
+ * An AbstractEvaluator analyzes the current system state and proposes an
+ * unordered list of TuningChoices that might improve the system performance.
+ */
+class AbstractEvaluator {
```

Comment:
Call this "AbstractTuningEvaluator" or something similar, more speaking. I have, e.g., a local "JoinOrderingEvaluator" that has nothing to do with this kind of evaluator you're describing here.

Reply:
renamed to `AbstractTuningEvaluator`